packages/api-sdk/src/medical/models/common/us-data.ts

```diff
@@ -53,4 +53,15 @@ export enum USState {
   WI = "WI",
   WY = "WY",
 }
-export const usStateSchema = z.nativeEnum(USState);
+
+const USStateMapping = Object.fromEntries(
```

Comment:
this mapping was necessary to satisfy typescript compiler that the output from the schema was not type string but actually type USState